src/Microsoft.ML.DataView/DataViewType.cs

```diff
@@ -473,7 +473,7 @@ public override bool Equals(DataViewType other)
     /// in <see cref="IDataView"/> would be the associated <see cref="DataViewType"/>.
     /// </summary>
     [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
-    public abstract class DataViewTypeAttribute : Attribute, IEquatable<DataViewTypeAttribute>
+    public abstract class DataViewTypeAttribute : TypeAttribute, IEquatable<DataViewTypeAttribute>
```

Comment:
>DataViewTypeAttribute [](start = 26, length = 21)

I think this change should work without the intermediate `TypeAttribute` class, by using `DataViewTypeAttribute` in `DataViewTypeManager` instead.

Reply:
So it looks like you are correct we can just use the DataViewTypeAttribute instead.